script.js

```diff
@@ -1,12 +1,421 @@
+    episodeImg.alt = `image`;
+
+    //take the summary of each episode
+    let episodeSummary = document.createElement(`article`);
+    let summaryArticle = containerDiv.appendChild(episodeSummary);
+    episodeSummary.setAttribute(
+      `id`,
+      `summaryS${codeCorrection(episode.season)}E${codeCorrection(
+        episode.number
+      )}`
+    );
+    episodeSummary.setAttribute(`class`, `summary`);
+
+    //summary tag inside article
+    let summaryTagP = document.createElement(`p`);
+    summaryArticle.appendChild(summaryTagP);
+    summaryTagP.setAttribute(
+      `id`,
+      `summaryTagS${codeCorrection(episode.season)}E${codeCorrection(
+        episode.number
+      )}`
+    );
+    summaryTagP.setAttribute(`class`, `summaryTag`);
+    summaryTagP.textContent = `Summary`;
+
+    //actual summary inside article
+    let summary = document.createElement(`p`);
+    summaryArticle.appendChild(summary);
+    summary.setAttribute(
+      `id`,
+      `summaryPS${codeCorrection(episode.season)}E${codeCorrection(
+        episode.number
+      )}`
+    );
+    summary.setAttribute(`class`, `summaryP`);
+    summary.textContent = episode.summary.replace(/(<([^>]+)>)/gi, "");
+  });
+}
+
+function makeShowPage(list) {
+  const rootElem = document.getElementById("root");
+  list.forEach((show) => {
+    let eachDiv = document.createElement(`div`);
+    let containerDiv = rootElem.appendChild(eachDiv);
+    containerDiv.setAttribute(`id`, `${show.id}`);
+    containerDiv.setAttribute(`class`, `container show`);
+    containerDiv.setAttribute(`style`, `display: initial;`);
+
+    //jump to episodes
+    containerDiv.addEventListener(`click`, () => {
+      let containerDiv = document.getElementsByClassName("container");
+      container = Array.from(containerDiv);
+      container.map((x) => (x.style.display = `none`));
+      setup(`https://api.tvmaze.com/shows/${show.id}/episodes`);
+      updateResult();
+    });
+
+    //take the name of each show and put it inside containerDiv
+    let nameParagraph = document.createElement(`p`);
+    containerDiv.appendChild(nameParagraph);
+    nameParagraph.setAttribute(`class`, `name`);
+    nameParagraph.textContent = `${show.name}`;
+
+    //film rating and runtime
+    let ratingP = paragraph();
+    containerDiv.appendChild(ratingP);
+    ratingP.setAttribute(`class`, `episodeCode rating`);
+    ratingP.textContent = `Average Rate:   ${show.rating.average}`;
+
+    //film runtime
+    let runtimeP = paragraph();
+    containerDiv.appendChild(runtimeP);
+    runtimeP.setAttribute(`class`, `episodeCode runtime`);
+    runtimeP.textContent = `Runtime:  ${show.runtime}`;
+
+    //film's genre
+    let genreP = paragraph();
+    containerDiv.appendChild(genreP);
+    genreP.setAttribute(`class`, `episodeCode runtime`);
+    genreP.id = `genre`;
+    genreP.textContent = `Genres:  ${show.genres}`;
+
+    //to fix positioning
+    let br = document.createElement(`br`);
```

Comment:
is this used?

Reply:
Bani Please put more comment on your code line by line would be better.

Reply:
> is this used?

oh, it is silly, but yes, because of changing the sizes of my images on hover it gave me a bit more space, just it.